tp_timesheet/__main__.py

```diff
@@ -33,7 +33,7 @@ def run():
     else:
         start_date = datetime.strptime(args.start, "%d/%m/%Y")
     dates = date_fn(start = start_date, count = args.count)
-    print("Date(s) to be submitted (YYYY-mm-dd):", [str(d) for d in dates])
+    print("Date(s) to be submitted (YYYY-mm-dd):", [str(DATE) for DATE in dates])
```

Comment:
```suggestion
    print("Date(s) to be submitted (YYYY-mm-dd):", [str(date) for date in dates])
```
All vars should be lowercase. Only exception is if they are global or named to reflect shell environment vars